### DISCUSSION TITLE
网易duilib控件是否可以直接从缓存中加载图片？

### DISCUSSION THREAD

**Discussion by QHHNB** [netease-im/NIM_Duilib_Framework#425] [Q&A]
我有一个相机，直接采集到了图片Mat，现在只发现duilib可以从某一个路径下面读取图片到控件，能否直接把mat存到缓冲中直接加载，因为先写到本地再加载路径下的图片有点慢，是否网易的duilib有提供某种方法实现这种功能？

**Comment by nmgwddj** [comment]
暂时没有直接的接口来处理，另外不确定的是你内存中的图片是什么格式的？

**Comment by QHHNB** [comment]
好的，我先研究一下setbkimage的底层实现

**Comment by QHHNB** [comment]
![image](https://user-images.githubusercontent.com/106065825/173202313-4e73704a-0a7b-4885-9a54-2a97e07ace8f.png)
已找到方法，duilib发现這個函數可以把bitmap存到内存里面，只需要把图片格式转为bitmap，然后自己指定路径，重新drawimage即可
